packages/vscode-extension/src/handlers.ts

```diff
@@ -670,80 +670,35 @@ export async function addCapabilityHandler(args?: any[]): Promise<Result<null, F
 }
 
 export async function addFeatureHandler(args?: any[]): Promise<Result<null, FxError>> {
-  // TODO: move the question model to fx-core addFeature API
-  const addFeatureConfig: SingleSelectConfig = {
-    name: "addFeature",
-    title: "Add features",
-    step: 1,
-    options: [
-      {
-        id: "capability",
-        label: `${localize("teamstoolkit.handlers.addFeature.capability.label")}`,
-        detail: localize("teamstoolkit.handlers.addFeature.capability.detail"),
-      },
-      {
-        id: "resource",
-        label: `${localize("teamstoolkit.handlers.addFeature.resource.label")}`,
-        detail: localize("teamstoolkit.handlers.addFeature.resource.detail"),
-      },
-      {
-        id: "additional",
-        label: `${localize("teamstoolkit.handlers.addFeature.additional.label")}`,
-        detail: localize("teamstoolkit.handlers.addFeature.additional.detail"),
-      },
-    ],
-  };
-  const additionalFeatureConfig: SingleSelectConfig = {
-    name: "additionalFeature",
-    title: "Additional features",
-    step: 2,
-    options: [
-      {
-        id: "sso",
-        label: `${localize("teamstoolkit.handlers.addFeature.sso.label")}`,
-        detail: localize("teamstoolkit.handlers.addFeature.sso.detail"),
-      },
-      {
-        id: "api",
-        label: `${localize("teamstoolkit.handlers.addFeature.api.label")}`,
-        detail: localize("teamstoolkit.handlers.addFeature.api.detail"),
-      },
-      {
-        id: "cicd",
-        label: `${localize("teamstoolkit.handlers.addFeature.cicd.label")}`,
-        detail: localize("teamstoolkit.handlers.addFeature.cicd.detail"),
-      },
-    ],
+  ExtTelemetry.sendTelemetryEvent(TelemetryEvent.AddCapStart, getTriggerFromProperty(args));
```

Comment:
What about add another 'AddFeature' event instead of reusing existing addcap event?

Reply:
Yes, missed this change. Will do.